spec/hardware/system.fmf

```diff
@@ -31,3 +38,8 @@ example:
     system:
         vendor-name: "~ HPE"
         numa-nodes: ">= 4"
+
+  - |
+    # Select a system that is not managed by deprecated lab controllers
+    system:
+        lab-controller: "!~ deprecated-lab-%.foo.bar.com"
```

Comment:
@happz do you think it makes sense to have it generic like this? if so I would rename it to `controller`, what might match some other infra and a similar concept.

Or maybe we should just put it under `beaker`, not spoil nice generic `system` with it?

Reply:
Yes, we have several options:

```yaml
controller-hostname: ...

beaker-controller-hostname: ...

beaker-lab-controller: ...

beaker:
  lab-controller: ....
```

Reply:
Or look at it from the point of view of a collocation, region, or zone:

```yaml
collocation:
  controller-hostname: ...?
  availability-zone: ...?
```

Reply:
I kinda like the last one, neat idea, that would match public clouds also beaker .... 

@lukaszachy @psss WDYT?

Reply:
@lkotek hey, any opinions, seems we are thinking about:

```
collocation:
  controller-hostname: ...?  (mapping to beaker lab-controller)
  availability-zone: ...? (mapping to AWS availability zone - example for the future)
```  

As the most probably alternative we would go with

Reply:
Hi @thrix, I think the last version is totally OK to me. Clear and easy to understand.

Reply:
Ack, having a generic section, not specific to `beaker`, sounds like a better, more clear approach. Regarding the name, `collocation` suggests that the values will be related to multihost tests only? I would say there might be a perfectly valid use case for single host execution as well. Perhaps `location` would fit a little bit more? Or shall we go with something even more generic like `general` as an umbrella section for various leftovers which do not fit anywhere else?

Reply:
> Regarding the name, `collocation` suggests that the values will be related to multihost tests only?

(I just noticed a persistent typo, it should be `colocation`, not `collocation`)

I would say so, it was supposed to be a label for properties related to where the machine is located - region, lab, continent, special connectivity, etc. As such, it is definitely applicable to single-host tests.

> Perhaps `location` would fit a little bit more?

Yes, `location` would fit the description as well. `colocation` came to my mind as a term used for server housing facilities, but the meaning is the same.

> Or shall we go with something even more generic like `general` as an umbrella section for various leftovers which do not fit anywhere else?

I'd try to avoid "general" sections as much as possible, and here we seem to have multiple properties we could host under this new section, i.e. it's not a lone left-over key, but a family of keys (`lab-controller`, possibly also `availability-zone`, `region`, ...)

Reply:
Changed to `location` and `location.lab-controller`.